src/main/java/ch/cern/eam/wshub/core/services/workorders/impl/LaborBookingServiceImpl.java

```diff
@@ -146,10 +144,15 @@ public String createLaborBooking(InforContext context, LaborBooking laborBooking
 								taskPlan.setCode(activity.getTaskCode());
 								taskPlan = taskPlanService.getTaskPlan(context, taskPlan);
 								WorkOrderActivityChecklistSignatureResult[] signatures = checklistService.getSignatures(context, workOrderNumber, activity.getActivityCode(), taskPlan);
-								if(signatures.length > 0) {
+								if (signatures.length > 0) {
 									activity.setChecklists(checklistService.readWorkOrderChecklists(context, activity));
-									if(taskPlan.getPerformedByRequired() || taskPlan.getReviewedByRequired()) {
-										activity.setSignatures(signatures);
+									if (taskPlan.getReviewedByRequired()) {
+										activity.setSignatures(Arrays.stream(signatures)
+												.collect(Collectors.toMap(WorkOrderActivityChecklistSignatureResult::getType, Function.identity())));
+									} else if (taskPlan.getPerformedByRequired()) {
+										activity.setSignatures(
+												Arrays.stream(signatures).filter(signature -> !signature.getType().equals("RB01"))
+														.collect(Collectors.toMap(WorkOrderActivityChecklistSignatureResult::getType, Function.identity())));
```

Comment:
If only Reviewed By Required is checked, a single performer signature is shown. Is this intended?